infra/aws/terraform/modules/eks-prow-iam/policies.tf

```diff
@@ -143,6 +143,8 @@ data "aws_iam_policy_document" "eks_apply" {
       "iam:TagPolicy",
       "iam:TagRole",
       "iam:UpdateOpenIDConnectProviderThumbprint",
+      "iam:PutRolePermissionsBoundary",
+      "iam:DeleteRolePermissionsBoundary",
```

Comment:
Should `DeleteRole*` go here or in the destroy policy?

Reply:
Good point